app/views/hub/efile_submissions/_efile_security_information.html.erb

```diff
@@ -0,0 +1,85 @@
+<% if @client.efile_security_informations.size.nonzero? %>
+  <div class="accordion spacing-above-10">
+    <a href="#" class="accordion__button" aria-expanded="true" aria-controls="a2">
+      <h3>Efile Security Information</h3>
+    </a>
+    <div class="accordion__content" id="a2">
+      <div class="client-container">
+        <div class="client-profile__column-container">
+          <div class="client-profile__column">
+            <div class="client-profile__field-group">
+              <% esf_1 = @client.efile_security_informations.first %>
+              <h2 class="text--bold"><%= "Efile Security Information (##{esf_1.id}) captured at #{timestamp(esf_1.created_at)}" %></h2>
+              <div class="field-display">
+                <span class="form-question">Browser Language:</span>
+                <span class="label-value"><%= esf_1.browser_language %></span>
+              </div>
+              <div class="field-display">
+                <span class="form-question">Client System Time:</span>
+                <span class="label-value"><%= esf_1.client_system_time %></span>
+              </div>
+              <div class="field-display">
+                <span class="form-question">IP Address:</span>
+                <span class="label-value"><%= esf_1.ip_address %></span>
+              </div>
+              <div class="field-display">
+                <span class="form-question">Platform:</span>
+                <span class="label-value"><%= esf_1.platform %></span>
+              </div>
+              <div class="field-display">
+                <span class="form-question">Timezone Offset:</span>
+                <span class="label-value"><%= esf_1.timezone_offset %></span>
+              </div>
+              <div class="field-display">
+                <span class="form-question">User Agent:</span>
+                <span class="label-value"><%= esf_1.user_agent %></span>
+              </div>
+              <div class="field-display">
+                <span class="form-question">Device ID:</span>
+                <span class="label-value"><%= esf_1.device_id %></span>
+              </div>
+            </div>
+          </div>
+          <% if @client.efile_security_informations.size > 1 %>
```

Comment:
I'm assuming we want to show the EfileSecurityInformation that we capture at the beginning and end of the flow

Reply:
Yes - good to show all of them!

Reply:
i'd honestly prefer that we loop through all of them and display all of them instead of limiting it to just two... but i guess i could be convinced otherwise.

I also think that this is a great candidate for a partial where we can pass the efile_security_information object in!

Reply:
@alaroia-cfa in the case that the first and last is not "all" of them, do you think we should be looping through all of them or just displaying the first and last.

Reply:
I think we should show all of them. We're showing these to support our own manual investigation, so I think it's good to show all of them.

Reply:
Thanks for the feedback y'all! Okay I can add a loop for all of them. I did two because I went into prod and queried for recent random clients and found that most of them had 1-2 associations to EfileSecurityInformation. And I didn't use a loop because I wanted them to be in two different columns. I'll take the accordion out of the partial too.

Reply:
@embarnard its possible to keep the column layout and loop! we can work on some css strategies for doing that this afternoon, if you want to pair on it for a bit!